README.md

```diff
@@ -91,23 +110,27 @@ Otherwise, you will need to speak to an admin who can force merge your full chan
 ### Building for Android
 Note there is an article written for how to build Holochain for Android, read it [here](doc/holochain_101/src/holochain_across_platforms.md).
 
-### Contribute
-We accept Pull Requests and welcome your participation. Please make sure to include the issue number your branch names and use descriptive commit messages.
+## Contribute
+Holochain is an open source project.  We welcome all sorts of participation and are actively working on increasing surface area to accept it.  Please see our [contributing guidelines](https://github.com/holochain/org/blob/master/CONTRIBUTING.md) for our general practices and protocols on participating in the community.
 
+### App Spec Driven Development
+In adding significant changes and new features to Holochain, we follow a specific test-driven development protocol:
+1. Start by creating a branch in the [app-spec-rust](https://github.com/holochain/app-spec-rust) repository which demonstrates an actual implementation of the use of the new feature in the sample application that lives in that repository, including tests that would pass if the feature were actually implemented here in the holochain-rust repo.
+1. Create a pull request on that branch for the development team to talk about and discuss the suggested change.  The PR triggers Continuous Integration tests which will initially fail, because they try and run the proposed changes against the `develop` branch of this `holochain-rust` repo.
+1. Do any development necessary in the `holochain-rust` and `hdk-rust` repos to implement the feature demonstrated in `app-spec-rust`
+1. Finally, when the feature is fully implemented, the CI tests should turn green on `app-spec-rust` and the branch can be merged indicating that that feature.
```

Comment:
Sentence trails off. Indicating that that feature... ?

Reply:
fixed